multi_map_it_test.go

```diff
@@ -619,3 +639,29 @@ func TestMultiMap_MultiMapEntryListener(t *testing.T) {
 		}
 	})
 }
+
+func TestMultiMap_GetObservation(t *testing.T) {
+	it.MultiMapTester(t, func(t *testing.T, m *hz.MultiMap) {
+		ctx := context.Background()
+		// make sure that underlying multi-map is empty
+		assert.EqualValues(t, 0, it.MustValue(m.Size(ctx)))
+		targetValues := map[string][]interface{}{
+			// represent nil return value
+			"nil": nil,
+			// represent return object with zero length and capacity
+			"object": make([]interface{}, 0),
+		}
```

Comment:
We don't need `targetValues`.

Reply:
https://github.com/hazelcast/hazelcast-go-client/pull/734/commits/8d5d17e46c48d1f3b6dbecd61ba9881f11c6a06a#diff-28f71a2bb2fe76407346964d0bd36f05083076e6a7af378c47b0fe9f37f8135eL665